app/cdap/components/WrangleHome/Components/OngoingDataExploration/__test__/OngoingDataExploration.test.tsx

```diff
@@ -14,25 +14,28 @@
  * the License.
  */
 
-import React from 'react';
-import { render, screen } from '@testing-library/react';
-import OngoingDataExploration from '../index';
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import OngoingDataExploration from "components/WrangleHome/Components/OngoingDataExploration/index";
 
-const testObj = {
-  connectionName: 'Upload',
-  workspaceName: 'Divami_Users_Emails.xlsx',
-  recipeSteps: 0,
-  dataQuality: 100,
-};
-
-test('renders Ongoing Data Exploration component', () => {
-  jest.mock('api/dataprep', () => {
-    return Promise.resolve([
-      { connectionName: 'yolo', workspaceName: 'Divami_Users_Emails.xlsx', recipeSteps: 0 },
-      { connectionName: 'Upload', workspaceName: 'Divami_Users_Emails.xlsx', recipeSteps: 0 },
-    ]);
+describe("It should test OngoingDataExploration Component", () => {
+  test("renders Ongoing Data Exploration component", () => {
+    jest.mock("api/dataprep", () => {
+      return Promise.resolve([
+        {
+          connectionName: "test1",
+          workspaceName: "Divami_Users_Emails.xlsx",
+          recipeSteps: 0,
+        },
+        {
+          connectionName: "Upload",
+          workspaceName: "Divami_Users_Emails.xlsx",
+          recipeSteps: 0,
+        },
```

Comment:
why do we need changes in ongoin data exploration test case file?
plz check and provide info

Reply:
@abhilashDivami

Reply:
@adityadivami  -- reverted changes done to OngoingDataExploration, after reverting , changed connectionName to "test1" from "YOLO"  while providing mock data to jest.Mock.